### DISCUSSION TITLE
Module needed?

### DISCUSSION THREAD

**Discussion by sascha284** [giovanni-bertoncelli/ngx-konva#27] [Q&A] [answered]
Hi, can you arrange your install information? What about todo if package is installed? Which module have to be integrated?

**Comment by ghost** [comment, marked as answer]
Of course you should import the Angular module 😉 . 
Take a look to the README: https://github.com/ctinnovation/ngx-konva?tab=readme-ov-file#installation, I've added some missing information about the installation and the package dependencies.
